website/docs/sidebars.js

```diff
@@ -4,7 +4,12 @@ module.exports = {
     'interactors/built-in-dom',
     'interactors/locators-filters-actions',
     'interactors/write-your-own',
-    'interactors/integrations'
+    'interactors/integrations',
+    {
+      "type": "category",
+      "label": "API",
+      "items": require('./sidebars/typedoc-interactor-sidebar')
```

Comment:
I'm curious to know how this is defined, could be an option to make things look more organized (I'm up for writing a manual sidebar for the Interactors API for instance)

Reply:
Mentioned in the previous comment, it is built by the plugins. I have to imagine there is some customizability of this through what typedoc outputs?